02-starting-with-data.Rmd

```diff
@@ -136,8 +137,27 @@ To open the dataset in RStudio's Data Viewer, use the `view()` function:
 view(surveys)
 ```
 
-> ### Note
->
+## Missing values in data
+Missing values are those not stored for a variable in the observation of interest. 
+This is a common problem across all domains of research. The reasons could be many. 
+Missing values may have a significant effect on the conclusions of the research. 
+When viewing above data, we observe a lot of cells showing ‘NA’ values. 
+Those are missing values. While care should be taken to collect data so that there 
+are no missing values, most of the times values go missing. 
+
+One of the ways to handle this is to omit those cases with the missing values. 
+Though, this would lead to a smaller sample size, it is unavoidable. 
```

Comment:
I'm not sure what you mean with "It is unavoidable". There are imputation techniques and modelling approaches (e.g. FIML) that can deal with missing data without reducing the sample size.